tests/cbtest/gtest/test_zrepl_prot.cc

```diff
@@ -218,11 +218,11 @@ class TestPool {
 	}
 
 	~TestPool() {
-		try {
+//		try {
 			execCmd("zpool", std::string("destroy -f ") + m_name);
-		} catch (std::runtime_error re) {
-			;
-		}
+//		} catch (std::runtime_error re) {
+//			;
+//		}
```

Comment:
I'm not sure if this is a good idea. But let's try it :-) In theory this command should never fail, but we know that even zpool destroy sometimes fails with EBUSY.

Reply:
this being test code, idea is - lets see if this errors out and fix the reason for it